recipe/meta.yaml

```diff
@@ -18,13 +18,13 @@ build:
 
 requirements:
   host:
-    - python >=3.7, <3.10
+    - python >=3.7
```

Comment:
This looks fine to me. Depending on how defensive you want to be, you could keep the upper bound and bump it to `<3.11` if you'd like. No strong thoughts from me one way or the other